spec/models/page_spec.rb

```diff
@@ -19,6 +19,22 @@
     end
   end
 
+  describe ".find_by_slug" do
+    it "finds a root-level page" do
+      pages = described_class.build_hierarchy(file_fixture("_pages").cleanpath)
```

Comment:
[suggestion, nonblocking] similar to pulling `pages` out of `subject`, you could pull `pages` up into a `let` and remove it from both of these tests:

```suggestion
  describe ".find_by_slug" do
    let(:pages) { described_class.build_hierarchy(file_fixture("_pages").cleanpath) }
    it "finds a root-level page" do
```